src/data/projects/newrelic-newrelic-python-agent.json

```diff
@@ -8,11 +8,11 @@
   },
   "title": "New Relic Python Agent",
   "supportUrl": null,
-  "defaultBranch": "main",
-  "contributingGuideUrl": "https://github.com/newrelic/newrelic-python-agent/blob/main/CONTRIBUTING.rst",
   "githubUrl": "https://github.com/newrelic/newrelic-python-agent",
   "permalink": "https://opensource.newrelic.com/projects/newrelic/newrelic-python-agent",
-  "iconUrl": null,
+  "defaultBranch": "main",
+  "contributingGuideUrl": https://github.com/newrelic/newrelic-python-agent/blob/main/CONTRIBUTING.rst,
+  "iconUrl": null 
```

Comment:
@carrieedwards, you're missing a comma at the end of line 15. It's preventing your build from completing (and me from merging your change). can you give me a fix here.